openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -243,6 +243,19 @@ message SKAdNetworkRequest {
 message ImpExt {
   // Publisher's SKAdNetwork information.
   optional SKAdNetworkRequest skadn = 1;
+
+  // To differentiate between impression opportunities
+  // that support Protected Audience API on-device auction from
+  // those that only support traditional server-side exchange auction
+  enum Ae {
```

Comment:
I recommend to use Google's schema for reference https://developers.google.com/authorized-buyers/rtb/downloads/open-beta/openrtb-adx-proto